### PR TITLE
Issue #13213: Remomve '//ok' comments from annotationlocation package…

### PR DESCRIPTION
**Pull request by Rohanraj123**
part of #13213 

Link to check the package : [annotationlocation](https://github.com/checkstyle/checkstyle/tree/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation)

### CODE REVIEW

**Review by romani — APPROVED** [review]
Ok to merge.

Thanks a lot for update. It was easy to review.

**Comment by romani** [comment]
@rnveach , @nrmancuso , I think we can merge update like this by single review.
Let me know if you are ok .

**Review by nrmancuso — CHANGES_REQUESTED** [review]
@Rohanraj123 we must remove associated suppressions, see https://github.com/checkstyle/checkstyle/pull/13609 as example. 

@romani as part of these PRs, we need to show proof of grep/ag on suppression file

**Comment by romani** [comment]
Thanks a lot I put your PR as example in issue description

**Comment by Rohanraj123** [comment]
@nrmancuso Sir, as per the directed suggestion i removed all the supressions related to annotation package.
Can you please review it.

**Comment by Rohanraj123** [comment]
@romani Sir some checks have failed .  Can you please review the changes i did manually or suggest me the way to qualify the failed checks.

**Comment by nrmancuso** [comment]
> @romani Sir some checks have failed . Can you please review the changes i did manually or suggest me the way to qualify the failed checks.

@Rohanraj123 `mvn clean verify` should always be passing on your local before pushing changes, please do this. Also, do not be afraid to click into failed builds and determine point of failure.

Also, please do 

>show proof of grep/ag on suppression file

as I have in example at https://github.com/checkstyle/checkstyle/pull/13609

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -139,14 +139,6 @@
...
 
   <!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationlocation[\\/]InputAnnotationLocationRecordsAndCompactCtors.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineRecordsAndCompactCtors.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineRecordsAndCompactCtors.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationusestyle[\\/]InputAnnotationUseStyleWithTrailingComma.java"/>
```
Comment (nrmancuso) [inline config/checkstyle-input-suppressions.xml:149]: Revert this change, these are not for the same check.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

**Comment by romani** [comment]
Always keep single commit , and do not do changes for unrelated files.
